pkg/config/config.go

```diff
@@ -126,6 +128,11 @@ func (cfg *Config) BindFlags() {
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.",
 	)
+	flag.BoolVar(
+		&cfg.EnableAdoptedResourceReconciler, flagEnableAdoptedResourceReconciler,
+		true,
+		"Enable the AdoptedResource reconciler.",
+	)
```

Comment:
Keeping the default to true for now. We should start thinking about deprecation and removal dates, once the adoption via annotation is support.